src/pages/feedback-analysis/styles/feedback-analysis.css

```
@@ -0,0 +1,736 @@
+  justify-content: space-between;
+  gap: 24px;
+}
+
+.feedback-analysis-page__banner-copy {
+  flex: 1;
+  min-width: 0;
+}
+
+.feedback-analysis-page__title {
+  margin: 0 0 6px;
+  font-family: "SUIT", "Pretendard", sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "SUIT" (font-family-name-quotes)

(font-family-name-quotes)